src/components/Player/Opsheet/Opsheet.js

```diff
@@ -59,7 +59,7 @@ class Opsheet extends Component {
           <button className='collapse-toggle' onClick={this.toggleSheetCollapseExpand}>+/-</button>
           
           <ListType />
-          <List listClassName = {this.listClassName}/> 
+          <List listClassName={this.state.listClassName}/> 
```

Comment:
This illustrates one reason I like destructuring.

If you had destructured:
```JavaScript
const {listClassName} = this;
```
in the first place, you could now change it to:
```JavaScript
const {listClassName} = this.state;
```
and anywhere you used `listClassName` would instantly be supplied with the new alternate value. Alternatively, if you had used this variable in several places, you'd have to go through and change it in every place.

Plus, destructuring at the top of your method allows someone who's reading your code from the top to the bottom to see what values should be found on which objects, and therefore which objects should be expected to have which values... and confidence that they're coming off the correct variables.

in other words, if you had used `this.props.listClassName` I wouldn't know if that's because you intended this to be a prop, or if you typo'd and meant to take the value off `state`.

But using `const {listClassName} = this.state;` is very "declarative" and sets up whoever is reading your code to understand what preconditions are necessary to use your function - as well as focusing your efforts by first setting up what variables should be available to you, and not having to remember what properties should be found on which objects, when you come back in a week to continue hacking on it.